olive/engine/engine.py

```diff
@@ -299,18 +299,24 @@ def run(
             self.footprints[accelerator_spec].record(model_id=input_model_id)
 
             try:
-                if evaluation_only:
+                if evaluate_input_model:
                     prefix_output_name = (
-                        f"{output_name}_{accelerator_spec}_" if output_name is not None else f"{accelerator_spec}_"
+                        f"{output_name}_{accelerator_spec}_" if output_name is not None else f"{accelerator_spec}"
                     )
-                    assert self.evaluator_config is not None, "Evaluation only is True but no evaluator provided"
+                    assert self.evaluator_config is not None, "evaluate_input_model is True but no evaluator provided"
                     results = self._evaluate_model(input_model, input_model_id, self.evaluator_config, accelerator_spec)
-                    result_name = f"{prefix_output_name}metrics"
+                    logger.info(f"Input model evaluation results: {results}")
+                    result_name = f"{prefix_output_name}_input_model_metrics"
                     results_path = output_dir / f"{result_name}.json"
                     with open(results_path, "w") as f:
                         json.dump(results.to_json(), f, indent=4)
+                    logger.info(f"Saved evaluation results of input model to {results_path}")
                     outputs[accelerator_spec] = results
-                elif self.no_search:
+                    if not self.passes:
```

Comment:
In this case, does the Olive only run for evaluating the input model?

Reply:
Yes, as there is no any passes were given.